frontend/packages/shared/src/constants.js

```diff
@@ -18,3 +18,5 @@ export const PROD_ENV_TYPE = 'production';
 export const PROTECTED_TASK_NAME_CUSTOM_RECEIPT = 'CustomReceipt';
 export const PREVIEW_MOCK_PARTY_ID = '51001';
 export const PREVIEW_MOCK_INSTANCE_GUID = 'f1e23d45-6789-1bcd-8c34-56789abcdef0';
+
+export const MEDIA_QUERY_MAX_WIDTH = '(max-width: 1000px)';
```

Comment:
Er `1000px` et tilfeldig valg, eller kommer det fra en gitt standard? I utgangspunktet tenker jeg det er mest ryddig å bruke de samme brekkpunktene som Apps, med forbehold om at jeg ikke har disse verdiene for hånden akkurat nå.

Reply:
1000px er ikke tilfeldig, men et tall gitt fra Anniken 😄

Reply:
@Annikenkbrathen, er dette en standard som vi følger på tvers av Altinn?

Reply:
Tja, jeg gjorde litt research og fant ut at det er vanlig å breake på 1000px. Jeg fant ingen standarder hos oss, så usikker på hva andre bruker. Enig i at det bær være det samme på tvers.

Designsystemet kommer til å anbefale en standard for breakpoints, men det har de ikke klart ennå. Så det kan være vi må rette opp i dette senere når de deler sin standard.
Spurte Lasse nå om han visste hva de kom til å anbefale og de tippet at de ville følge denne om det ikke var en grunn for noe annet https://aksel.nav.no/grunnleggende/styling/brekkpunkter

Kanskje vi skal bruke den vi også da?

Reply:
@Annikenkbrathen dette er vel også noe vi kan ta med inn til design på tvers i Altinn, for å dokumentere et mønster på dette? 😊

Reply:
Absolutt! kan legge det inn i boardet for design decision forum

Reply:
Skal jeg endre brekkpunktet til 1024 da? 😄